waterloop-site/src/components/SustainableTech/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+import IconBlackImg from '../../../static/img/sustainable-tech/Icon_Black.svg';
+import scrollTo from '../utils/LinkScroll';
+
+const NavbarContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 64px;
+  background-color: #e5f6fa;
+  position: fixed;
+  top: 0;
+  scroll-behavior: smooth;
```

Comment:
I tested out the website on Safari and it seems that the `scroll-behaviour` property doesn't work. The scroll to function still works but it doesn't have the smooth effect on Safari. I think this link has a [fix](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo#browser_compatibility)

Reply:
Hey, actually, that doesn't have any impact on the actual scroll behvaiour, that specifically belongs to the scrollTo function in the utils folder.
![image](https://user-images.githubusercontent.com/71536798/157275887-b52d8ecc-ad01-4376-a2be-fff22cf35056.png)

Reply:
Looking further into it, it seems like there's a potential solution by inputting a polyfill, but I don't think we should be adding another dependency just for this.

https://stackoverflow.com/questions/51229742/javascript-window-scroll-behavior-smooth-not-working-in-safari

Reply:
That's a good point! I think it should be fine then since the navigation still works well.

Reply:
Oh shoot I didn't know Safari doesn't support it :/ Yeah I would be reluctant to install a new dependency just for this, we can maybe look into a different implementation of the scroll functionality using jquery. Since the other browsers seem to be supported for now though, I think we can address this a bit later, thanks for bringing it up though Vanessa! and thanks for looking into it Evan :)